kalite/distributed/static/js/distributed/topics/views.js

```diff
@@ -20,7 +20,6 @@ window.ContentAreaView = BaseView.extend({
     show_view: function(view) {
 
         // hide any messages being shown for the old view
```

Comment:
Yeah, in a parallel PR this got replaced by the weird, and what I think is unnecessary clearAllMessages function (which appears to just do the same as clear_messages, but without the capacity to set an argument for message type).